### PR TITLE
Improve examples in README.md

### PR DESCRIPTION
**Pull request by PatrickCronin**
This PR seeks to improve examples provided.

Firstly, many of the examples currently use the test databases at https://github.com/maxmind/MaxMind-DB. Since many of our customers don't use these, we thought updating the examples with databases our customers are more likely to use will be helpful. Also along these lines, we moved the more common databases higher up in the examples to increase familiarity.

Secondly, we add some additional examples using the `jq` utility to parse the output of various databases and format it in different ways.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -313,57 +344,57 @@ $ mmdbinspect -db GeoIP2-Connection-Type-Test.mmdb 1.0.9.16/28 2003::ff00:3a4c/1
     <summary>Look up multiple IPs/networks in multiple databases</summary>
 
 ```bash
-$ mmdbinspect -db GeoIP2-Static-IP-Score-Test.mmdb -db GeoIP2-User-Count-Test.mmdb 1.2.3.5 1.2.3.66
+$ $ mmdbinspect -db /usr/local/share/GeoIP/GeoIP2-Static-IP-Score.mmdb -db /usr/local/share/GeoIP/GeoIP2-User-Count.mmdb 72.10.10.10 73.11.11.11
```
Comment (horgh) [inline README.md:347]: There's an extra $ here. And do you intend to show the full path in this case?

File: README.md [hunk 2]
```diff
@@ -95,26 +95,64 @@ This installs `mmdbinspect` to `$GOPATH/bin/mmdbinspect`.
 
 ## Examples
 
-The following examples were created using our test databases which can be found in `test/data` in this repository.
```
Comment (horgh) [inline README.md:98]: Do we want to keep that directory now?

Comment (oalders) [reply README.md:98]: Wouldn't that break the tests? https://github.com/maxmind/mmdbinspect/blob/master/pkg/mmdbinspect/mmdbinspect_test.go#L10-L11

File: README.md [hunk 3]
```diff
@@ -423,9 +423,48 @@ $ cat list.txt | xargs mmdbinspect -db GeoIP2-ISP.mmdb
...
+
+Print out the `city` and `country` names from each record using array construction and `join`:
+```bash
+$ mmdbinspect -db GeoIP2-City.mmdb 2610:30::/64 | jq -r '.[].Records[].Record | [.city.names.en, .country.names.en] | join(", ")'
+Martinsburg, United States
+```
+
+Get the AS number for an IP:
+```bash
+mmdbinspect -db GeoLite2-ASN.mmdb 152.216.7.49 | jq -r '.[].Records[].Record.autonomous_system_number'
```
Comment (horgh) [inline README.md:452]: Should this one have $ at the beginning like the rest?

**Pull request merged**